src/index.js

```diff
@@ -1,35 +1,163 @@
+/* eslint-disable eqeqeq */
```

Comment:
- [ ] Please don't disable linter rules. These are here to write clean and standard code.